storybook/webpack.config.js

```diff
@@ -1,86 +1,61 @@
 const path = require('path');
 const pkg = require('../package.json');
+const { readdirSync, statSync } = require('fs');
+const { parse } = require('path');
 
-module.exports = {
-  module: {
-    rules: [
-      // Storysource Addon
-      {
-        test: /\.stories\.jsx?$/,
-        loaders: [require.resolve('@storybook/addon-storysource/loader')],
-        enforce: 'pre'
-      },
-      // Css
-      {
-        test: /\.css$/,
-        loaders: ['style-loader', 'css-loader'],
-        include: [
-          path.resolve(__dirname, '../src'),
-          path.resolve(__dirname, './')
-        ]
-      },
-      // Sass
-      {
-        test: /\.scss$/,
-        use: [
-          { loader: 'style-loader' },
-          { loader: 'css-loader' },
-          {
-            loader: 'sass-loader',
-            options: {
-              includePaths: [
-                path.resolve(__dirname, '../sass/patternfly-react'),
-                ...Object.values(pkg.sassIncludes).map(includePath =>
-                  path.resolve(__dirname, `../${includePath}`)
-                )
-              ]
-            }
+const ROOT_DIR = '..';
+const PCKGS = `${ROOT_DIR}/packages`;
+
+const packages = readdirSync(path.resolve(__dirname, PCKGS))
```

Comment:
this can actually be replaced with lernas package utils
```
const PackageUtilities = require('lerna/lib/PackageUtilities');
const Repository = require('lerna/lib/Repository');

const packages = PackageUtilities.getPackages(new Repository());
```
Each "package" includes the properties here: https://github.com/lerna/lerna/blob/master/core/package/index.js#L36